app/src/main/java/com/ntg/user/mvpsample/data/sourse/remote/RemoteTaskRepo.java

```diff
@@ -0,0 +1,66 @@
+package com.ntg.user.mvpsample.data.sourse.remote;
+
+import com.ntg.user.mvpsample.data.Task;
+import com.ntg.user.mvpsample.data.sourse.TasksDataSource;
+import java.util.ArrayList;
+import java.util.List;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class RemoteTaskRepo implements TasksDataSource {
```

Comment:
it's better to have repo for each module (save/get/show details) , we want to have 1 view --> 1 presenter --> 1 repo